config/initializers/json.rb

```diff
@@ -0,0 +1,9 @@
+# monkey patch for the json gem to fix ruby keyword arg deprecation
+# https://github.com/flori/json/issues/399
+module JSON
+  module_function
+
+  def parse(source, opts = {})
+    Parser.new(source, **opts).parse
+  end
+end
```

Comment:
I added this to fix ruby deprecation warning. The code is in the gem. See https://github.com/flori/json/issues/399

Reply:
ah. interesting.